webapp/src/Controller/Jury/ExecutableController.php

```diff
@@ -39,9 +40,22 @@ public function __construct(
         protected readonly EventLogService $eventLogService
     ) {}
 
+    private function checkDisabled(Executable $e): bool
```

Comment:
This should be `checkEnabled` if it returns true when enabled. Also maybe move this to the executable.php file?